### ISSUE TITLE
Performance improvement

### ISSUE THREAD

**Issue by greenfork** [lePerdu/kakboard#8]
https://github.com/lePerdu/kakboard/blob/4cc87b2e2a1cb1e8d61d36de172751e874ff9300/kakboard.kak#L153-L158

This and the next loop add about 50ms total to startup time. If we define these commands manually using `map`: `str-list kakboard_paste_keys p P R <a-p> <a-P> <a-R>` then it goes down to under 5ms.

I'm not asking for anything in particular, but this might be useful to someone.

My config for reference:
```
set global kakboard_copy_cmd 'xclip -in -selection clipboard'
set global kakboard_paste_cmd 'xclip -out -selection clipboard'

map global normal y ': kakboard-with-push-clipboard y<ret>'
map global normal c ': kakboard-with-push-clipboard c<ret>'
map global normal d ': kakboard-with-push-clipboard d<ret>'

map global normal p ': kakboard-with-pull-clipboard p<ret>'
map global normal P ': kakboard-with-pull-clipboard P<ret>'
map global normal R ': kakboard-with-pull-clipboard R<ret>'
```

**Comment by lePerdu** [comment]
Yeah I cringe a little every time I remember that Kakoune constantly runs a ton of shell commands at startup and on every keypress. I don't really want to remove `kakboard-enable/disable` since it would require extra user configuration (the code shown above), but this is definitely a good option if you want to improve performance at the cost of some more explicit and repetitive configuration.

P.S. @greenfork Is there an easy and fancy way to measure Kakoune's performance, or are you just using a debugger / shell commands?

**Comment by greenfork** [comment]
Nothing fancy really, I run time measurement of kakoune startup in a loop and edit config, as a result it shows changes in real time. I use [elvish](https://elv.sh/) as a shell and for me the command is 

```
while true { -time { kak -e quit }; sleep 1 }
```

and then I just edit config, save and see if it changes.

About all the extra shell commands on init and keypress: I managed to reduce init time to under 100ms so it's generally fine. Mostly by removing `plug` for loading modules, I use it just to download and upgrade plugins. And keypresses - I don't notice anything significant there too, but I may be running a comparatively lightweight config in this regard.
